commandapi-platforms/commandapi-bukkit/commandapi-bukkit-core/src/main/java/dev/jorel/commandapi/CommandAPIBukkit.java

```diff
@@ -378,10 +378,18 @@ public void preCommandRegistration(String commandName) {
 		// Warn if the command we're registering already exists in this plugin's
 		// plugin.yml file
 		final PluginCommand pluginCommand = Bukkit.getPluginCommand(commandName);
-		if (pluginCommand != null) {
+		if (pluginCommand == null) {
+			return;
+		}
+		String pluginName = pluginCommand.getPlugin().getName();
+		if (config.getPlugin().getName().equals(pluginName)) {
 			CommandAPI.logWarning(
 				"Plugin command /%s is registered by Bukkit (%s). Did you forget to remove this from your plugin.yml file?"
-					.formatted(commandName, pluginCommand.getPlugin().getName()));
+					.formatted(commandName, pluginName));
+		} else {
+			CommandAPI.logNormal(
+				"Plugin command /%s is registered by Bukkit. You may have to use /minecraft:%s to execute your command."
```

Comment:
Is there any particular reason why we're not including the plugin name here too?

Reply:
No, there isn't really a reason. Should I add the plugin name?

Reply:
Yes please :)